tox.ini

```diff
@@ -29,7 +29,7 @@ deps =
     oracle: cx_Oracle
     postgres: psycopg2
 commands =
-    coverage run manage.py test {posargs}
+    python -W error::DeprecationWarning -W error::PendingDeprecationWarning -m coverage run manage.py test {posargs}
```

Comment:
This is copying the same pattern I use on my repos, to avoid deprecation warnings being triggered in a package.